packages/plugins/src/foundry/utils/index.ts

```diff
@@ -1203,13 +1206,16 @@ export const assertSphinxFoundryForkInstalled = async (
     throw new Error(`Foundry is not installed.`)
   }
```

Comment:
I don't think we need to check this anymore. Previously, this function used `forge --version` to get the commit hash, which isn't used anymore